Atomic/util.py

```diff
@@ -447,7 +445,7 @@ def default_docker():
 def default_docker_lib():
     try:
         return get_docker_conf()["DockerRootDir"]
-    except ValueError:
+    except (NoDockerDaemon, requests.ConnectionError):
```

Comment:
We want to make atomic agnostic to container runtime.  Do not want it complaining about docker not running.